Framework/Core/src/AnalysisDataModel.cxx

```diff
@@ -0,0 +1,111 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#include "Framework/AnalysisDataModel.h"
+#include "Framework/StringHelpers.h"
+#include "Framework/Logger.h"
+#include <boost/algorithm/string.hpp>
+
+namespace o2
```

Comment:
You can inline this, `namespace o2::aod::datamodel {}`

Reply:
done